setup.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import ast
```

Comment:
Unused import
```suggestion
```

Reply:
Oops, that was left from doing an `ast.literal_eval` on the `os.getenv` but I removed it to avoid any tracebacks that could come from invalid data.

Reply:
Still present ;-)